server/services/caseloadService.ts

```diff
@@ -104,7 +104,21 @@ export default class CaseloadService {
       })
 
     const [withLicence, pending] = await Promise.all([casesWithLicences, casesPendingLicence])
-    const casesWithComs = await this.mapResponsibleComsToCasesWithExclusions(withLicence.concat(pending))
+
+    const allCases = withLicence.concat(pending)
+    const allCasesWithStatusOverrides = allCases.map(c => {
+      const managedCase = c
```

Comment:
I find this declaration weird and unnecessary, but eslint didn't like reassignment of an existing param